x-pack/legacy/plugins/lens/public/xy_visualization_plugin/xy_suggestions.ts

```diff
@@ -146,30 +175,68 @@ function getSuggestion(
     return buildSuggestion(options);
   }
 
+  const sameStateSuggestions: Array<VisualizationSuggestion<State>> = [];
+
   // if current state is using the same data, suggest same chart with different presentational configuration
 
   if (xValue.operation.scale === 'ordinal') {
     // flip between horizontal/vertical for ordinal scales
-    return buildSuggestion({
-      ...options,
-      title: i18n.translate('xpack.lens.xySuggestions.flipTitle', { defaultMessage: 'Flip' }),
-      isHorizontal: !options.isHorizontal,
-    });
+    sameStateSuggestions.push(
+      buildSuggestion({
+        ...options,
+        title: i18n.translate('xpack.lens.xySuggestions.flipTitle', { defaultMessage: 'Flip' }),
+        isHorizontal: !options.isHorizontal,
+      })
+    );
+  } else {
+    // change chart type for interval or ratio scales on x axis
+    const newSeriesType = flipSeriesType(seriesType);
+    sameStateSuggestions.push(
+      buildSuggestion({
+        ...options,
+        seriesType: newSeriesType,
+        title: newSeriesType.startsWith('area')
+          ? i18n.translate('xpack.lens.xySuggestions.areaChartTitle', {
+              defaultMessage: 'Area chart',
+            })
+          : i18n.translate('xpack.lens.xySuggestions.barChartTitle', {
+              defaultMessage: 'Bar chart',
+            }),
+      })
+    );
   }
 
-  // change chart type for interval or ratio scales on x axis
-  const newSeriesType = flipSeriesType(seriesType);
-  return buildSuggestion({
-    ...options,
-    seriesType: newSeriesType,
-    title: newSeriesType.startsWith('area')
-      ? i18n.translate('xpack.lens.xySuggestions.areaChartTitle', {
-          defaultMessage: 'Area chart',
-        })
-      : i18n.translate('xpack.lens.xySuggestions.barChartTitle', {
-          defaultMessage: 'Bar chart',
-        }),
-  });
+  // flip between stacked/unstacked
+  sameStateSuggestions.push(
+    buildSuggestion({
+      ...options,
+      seriesType: toggleStackSeriesType(seriesType),
+      title: seriesType.endsWith('stacked')
+        ? i18n.translate('xpack.lens.xySuggestions.unstackedChartTitle', {
+            defaultMessage: 'Unstack',
```

Comment:
I found this wording confusing because the lines in the preview are so tiny they're almost invisible. What about the phrase "Side-by-side" instead of "Unstack"? If that's too complicated, I think both of these labels should be adjectives instead of noun. "Stacked" and "Unstacked" would be the smallest change.

Reply:
They are meant to be verbs to be consisted with `Flip`, but I agree that we can do a better job here. I will rename them to Stacked and Unstacked for now, we are planning to do a pass over labels before the release anyway.